activity-stream/simple-snippet.html

```diff
@@ -119,11 +119,33 @@ <h1>{{ title }}</h1>
   </section>
 
   {% if button_label and button_url %}
-    <a class="button-link" href="{{ button_url|safe }}">{{ button_label }}</a>
+    <a class="button-link" href="{{ button_url|safe }}" id="button">{{ button_label }}</a>
```

Comment:
Since this `a` it only needs `data-metric="button-click"` and the generic code handles that, otherwise we send double metrics.